std_msgs/Header header
string          interface_name
float32         board_voltage
int32           status_flag
string          status
uint8           motor_num
float32         velocity
int32           position
int32           torque